### PR TITLE
Replace linq-method .

### PR DESCRIPTION
**Pull request by ly3027929699**
I don't like linq,Including many game developers I know, we all feel that it allocates too much managed momeory

### CODE REVIEW

File: Doprez.Stride/Extensions/EntityComponentExtensions.cs [hunk 1]
```diff
@@ -13,30 +13,56 @@ public static class EntityComponentExtensions
     /// <typeparam name="T"></typeparam>
     /// <param name="entity"></param>
     /// <returns></returns>
-    public static T GetComponent<T>(this Entity entity)
+    public static T GetComponent<T>(this Entity entity) where T : EntityComponent
```
Comment (Doprez) [inline Doprez.Stride/Extensions/EntityComponentExtensions.cs:16]: The change for Linq is completely understandable. Although I dont want to add the requirement of EntityComponent here. The reason I made this method was due to the limitation of not getting Interface types from Entities compared to the existing Get<T>() from Strides default method.

**Review by Doprez — COMMENTED** [review]
Overall these changes make sense (naming was pretty lazy from me lol) 

The only concern is the requirement of EntityComponent. Maybe a better named method could be made but these were meant as alternatives to the default Get<T>() method from Stride.

File: Doprez.Stride/Extensions/EntityComponentExtensions.cs (hunk 1 above)
Comment (ly3027929699) [reply Doprez.Stride/Extensions/EntityComponentExtensions.cs]: I removed the generic limits

**Review by Doprez — APPROVED** [review]
Thank you!

**Pull request merged**